src/devices_models/devices/hydro_generation.jl

```diff
@@ -545,13 +547,13 @@ function nodal_expression!(
         include_parameters(
             psi_container,
             ts_data_active,
-            UpdateRef{H}(ACTIVE_POWER, "get_rating"),  # TODO: fix in PR #316
+            UpdateRef{H}(ACTIVE_POWER, "get_activepower"),
             :nodal_balance_active,
         )
         include_parameters(
             psi_container,
             ts_data_reactive,
-            UpdateRef{H}(REACTIVE_POWER, "get_rating"),  # TODO: fix in PR #316
+            UpdateRef{H}(REACTIVE_POWER, "get_reactivepower"),
```

Comment:
This doesn't apply here see above comment

Reply:
I don't follow

Reply:
Hydro doesn't have limitations on reactive power based on time-series. There isn't a reason to create a parameter here.